File: FlouraBackend/Floura.Core/Story.cs
using System;

public class Story
{
    public string? Title { get; set; }
    public string? Summary { get; set; }

    public bool IsValid()
    {
        return !string.IsNullOrWhiteSpace(Title)
            && !string.IsNullOrWhiteSpace(Summary);
    }
}

File: FlouraFrontend/FlouraMobileApp/src/models/Story.ts
export interface Story {
    id: string;
    title: string;
    summary: string;
    coverImage: string;
    backgroundImageKey?: string;
    storyBits: StoryBit[];
    ageRange: string;
}

export interface StoryBit {
    id: string;
    content: string;
    imageUrl?: string;
    order: number;
}